test_delete_all_tasks.py

```diff
@@ -0,0 +1,145 @@
+from todo import TodoList
+from pytest import fixture
+
+#we need minimal tests for adding, since there are other test files for them. 
+#i think testing a normal task, and an empty one would be sufficient
+@fixture 
+def homework_task():
+    return "do homework"
+
+@fixture 
+def empty_task():
+    return ""
+
+@fixture 
+def empty_task_list(empty_task):
+    todo_list = TodoList()
+    todo_list.add_task(empty_task)
+    return todo_list
+
+@fixture 
+def no_tasks_list():
+    return TodoList()
+
+@fixture 
+def one_task_list(homework_task):
+    todo_list = TodoList()
+    todo_list.add_task(homework_task)
+    return todo_list
+
+@fixture
+def two_task_list(homework_task):
+    todo_list = TodoList()
+    todo_list.add_task(homework_task)
+    todo_list.add_task(homework_task)
```

Comment:
The add_task method on lines 25 through 27 of TodoList.py checks for duplicate tasks and returns without adding a task if the duplicate exists. All tests involving this method don't only have one task in the list at the time of deletion so aren't properly testing the multiple tasks case.